jqm-all/jqm-engine/src/main/java/com/enioka/jqm/engine/JqmEngine.java

```diff
@@ -245,6 +250,16 @@ public void start(String nodeName, JqmEngineHandler h)
             // Pollers
             syncPollers(cnx, this.node);
             jqmlogger.info("All required queues are now polled");
+
+            // Standalone sequence reinit
+            final var standaloneMode = Boolean.parseBoolean(
+                GlobalParameter.getParameter(DbManager.getDb().getConn(), "wsStandaloneMode", "false"));
+            final var localIp = Inet4Address.getLocalHost().getHostAddress();
+            if (standaloneMode) {
```

Comment:
May be add a check if standalone and db != from hsqldb throw error not implemented ?

Reply:
Pas faisable proprement, car on ne connait l'etat du flag standalone qu'une fois la DB active.